modules/efk-logging-fluentd-log-rotation.adoc

```diff
@@ -19,23 +19,34 @@ fluentd.log.3  1Mb
 fluentd.log.4  1Mb
 ----
 
+The following example shows the same conditions for Rsyslog.
+
+----
+rsyslog.log     0b
+rsyslog.log.1  1Mb
+rsyslog.log.2  1Mb
+rsyslog.log.3  1Mb
+rsyslog.log.4  1Mb
+----
+
 .Prerequisites
 
 Set cluster logging to the unmanaged state.
 
 .Procedure
 
-You can control the size of the Fluentd log files and how many of the renamed files that {product-title} retains using
+You can control the size of the log collector files and how many of the renamed files that {product-title} retains using
 environment variables.
 
-.Parameters for configuring Fluentd log rotation
+.Parameters for configuring log rotation
```

Comment:
The log rotation of rsyslog is different from fluentd. There has a configmap logrotate-crontab controlls the rsyslog's log rotation.  You can get some information from: https://bugzilla.redhat.com/show_bug.cgi?id=1730211#c1